eng/common/templates/job/job.yml

```diff
@@ -101,11 +101,12 @@ jobs:
     # handle key-value variable syntax.
     # example:
     # - [key]: [value]
-    - ${{ if and(eq(variable.name, ''), eq(variable.group, '')) }}:
+    - ${{ if and(eq(variable.name, ''), eq(variable.group, ''), eq(variable.template, '')) }}:
```

Comment:
I'll revert the changes in this file after CI completes so I can limit re-triggering CI.